packages/components/psammead-podcast-promo/src/components/card.jsx

```diff
@@ -1,11 +1,12 @@
 import styled from '@emotion/styled';
-import { C_GHOST } from '@bbc/psammead-styles/colours';
+import { C_GHOST, C_EBON } from '@bbc/psammead-styles/colours';
 import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 
 const Card = styled.div`
   position: relative;
   background-color: ${C_GHOST};
   display: flex;
+  box-shadow: 0 0 0.3125rem 0.3125rem ${C_EBON}08;
```

Comment:
is the `08` at the end supposed to be there?

Reply:
Haha. I defer to https://github.com/bbc/psammead/pull/4214#discussion_r555677574

It's adding a 0.03 transparency to the `${C_EBON}` hash value